scalding-core/src/main/scala/com/twitter/scalding/graph/Edge.scala

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2012 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package com.twitter.scalding.graph
+
+case class Edge[T: Ordering, S](source: T, dest: T, attr: S)
```

Comment:
I'm a bit worried about `T: Ordering` here. This will have the `Ordering` serialized with each edge, sadly. Can we move the `T: Ordering` to methods that actually require an ordering?